config/locales/en/course/assessment/question/text_response.yml

```diff
@@ -4,14 +4,15 @@ en:
       question:
         text_responses:
           new:
-            header: 'New Text Response Question'
+            text_response: 'New Text Response Question'
+            file_upload: 'New File Upload Question'
           create:
-            success: 'The text response question was created.'
+            success: 'The %{name} was created.'
```

Comment:
I got the flash message when I checked out your branch and tried it:

<img width="522" alt="screen shot 2016-09-23 at 5 06 45 pm" src="https://cloud.githubusercontent.com/assets/6252919/18780450/39492fca-81b0-11e6-8f28-50c542450add.png">